src/consensus/aft/request.h

```diff
@@ -13,19 +13,18 @@ namespace aft
 {
   struct Request
   {
-    uint64_t caller_id;
     kv::TxHistory::RequestID rid;
     std::vector<uint8_t> caller_cert;
     std::vector<uint8_t> raw;
     uint8_t frame_format = enclave::FrameFormat::http;
 
-    MSGPACK_DEFINE(caller_id, rid, caller_cert, raw, frame_format);
+    MSGPACK_DEFINE(rid, caller_cert, raw, frame_format);
```

Comment:
Knock-on effect - `caller_id` isn't guaranteed to exist, so isn't passed through `aft::Request` (and since nothing broke, nothing was actually using it). This ID (or another kind of identity) can (and should?) be derived indepdently, from the original request, on each executor.